src/dotnet/ToolPackage/IProjectRestorer.cs

```diff
@@ -13,6 +13,7 @@ internal interface IProjectRestorer
             DirectoryPath assetJsonOutput,
             FilePath? nugetConfig = null,
             string source = null,
-            string verbosity = null);
+            string verbosity = null,
+            DirectoryPath? nugetCacheLocation = null);
```

Comment:
Test induced design damage. However, I don't know how to test it goes to nuget cache.